### ISSUE TITLE
Code block indentation spacing is too small.

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/spectacle#898]
See: https://github.com/FormidableLabs/spectacle/issues/886#issuecomment-619770673

**Actual Code**:
![Screenshot 2020-04-27 at 08 55 15](https://user-images.githubusercontent.com/8681413/80342686-dcc18a80-8864-11ea-8ae7-9143e80df58f.png)

**Presentation**:
<img width="329" alt="Screenshot 2020-04-27 at 08 55 22" src="https://user-images.githubusercontent.com/8681413/80342697-e2b76b80-8864-11ea-94b9-dcbfde3b5259.png">

I've seen this too in private presentations using MDX.

**Comment by LongLiveCHIEF** [comment]
I'm getting this when using JSX, so I don't think it's just md or MDX.  even when using `indentSize` prop.  I also tried switching my `CodePane` content so it was using tabs instead of spaces, and it still trimmed all tab characters and whitespace upon presentation.

**Comment by ryan-roemer** [comment]
@LongLiveCHIEF -- Is it trimming them, or are they just really, really small? I ended up doing like 8x the normal spaces I wanted then, finally saw them. (My strong hunch is that it's styling related)

**Comment by LongLiveCHIEF** [comment]
I'll check back in the morning and let you know.

**Comment by LongLiveCHIEF** [comment]
@ryan-roemer it's actually a combination of both.  There is no whitespace or html space characters in the markup, and the padding is 5px + 0.25em of the flex container.

**Comment by LongLiveCHIEF** [comment]
Whitespace/tab characters shouldn't be stripped, because they need to be there for copy/paste capabilities for the presentation visitors to copy code into their IDE's without it looking like a mess.

**Comment by ryan-roemer** [comment]
@LongLiveCHIEF -- My understanding is that the whitespace is "translated" by the code formatter to then a style amount, so it's not "stripped" per se, it's just "used for math" to do the style part.

Let me rephrase my question -- if you do 8x the normal whitespacing in code samples, do you visually see _more_ indentation or absolutely no difference in slide results?

**Comment by ryan-roemer** [comment]
I've captured "copy and pasteable code blocks" separately as a feature request in: https://github.com/FormidableLabs/spectacle/issues/928

**Comment by ryan-roemer** [comment]
*Update*: current WIP PR fixes this with correct spacing (and it's real spaces too) in MD -- https://github.com/FormidableLabs/spectacle/pull/934

![Screen Shot 2020-07-28 at 11 05 33 AM](https://user-images.githubusercontent.com/445557/88704194-9b8a8200-d0c2-11ea-9691-98e1ffbf91e5.png)
